lib/live_cluster/client/types.py

```diff
@@ -248,3 +249,6 @@ class ASInfoNotAuthenticatedError(ASInfoError):
 
 class ASInfoClusterStableError(ASInfoError):
     pass
+
+
+SIndexBinType = Union[Literal["NUMERIC"], Literal["GEO2DSPHERE"], Literal["STRING"]]
```

Comment:
Where is this used? Is it needed?

Reply:
It is not used.  I was trying to use it somewhere, but it caused some IntelliSense problems.  It is just a typing